tools/serverpod_cli/lib/src/test_util/builders/enum_definition_builder.dart

```diff
@@ -30,7 +31,8 @@ class EnumDefinitionBuilder {
           ProtocolEnumValueDefinition('B'),
           ProtocolEnumValueDefinition('C'),
         ],
-        _documentation = [];
+        _documentation = [],
+        _type = TypeDefinitionBuilder().withClassName('Example').build();
```

Comment:
Suggestion: This type does not have the enum definition on it, make sure that the builder represents the real state valid state by default!